### ISSUE TITLE
ParmEd/AmberTools issues in CI

### ISSUE THREAD

**Issue by mattwthompson** [openforcefield/openff-toolkit#1593]
https://github.com/openforcefield/openff-toolkit/actions/runs/4805975300/jobs/8552942734

#1532

**Comment by mattwthompson** [comment]
AFAICT this is only an issue for our testing infrastructure as we do not directly import ParmEd in the source code (but the examples do use it).

```shell
$ grep -r parmed openff                                                         8:51:20  ☁  fix-1593 ☂ ✭
openff/toolkit/tests/test_forcefield.py:        import parmed
openff/toolkit/tests/test_forcefield.py:            amber_parm = parmed.load_file(top_filepath, crd_filepath)
openff/toolkit/tests/test_forcefield.py:            structures.append(amber_parm.copy(parmed.Structure))
openff/toolkit/tests/test_forcefield.py:        import parmed as pmd
openff/toolkit/tests/test_forcefield.py:        import parmed as pmd
openff/toolkit/data/reference_energies/reference_after_802.json.txt:parmed                    at20RC5+54.g5702a232fe.dirty          pypi_0    pypi
openff/toolkit/data/reference_energies/reference_0.8.0.json.txt:parmed                    at20RC5+54.g5702a232fe.dirty          pypi_0    pypi
openff/toolkit/data/reference_energies/reference_after_1007.json.txt:  - parmed=3.4.0=py38h91a8764_1

**Comment by j-wags** [comment]
While the underlying incompatibility remains, it's now documented as pins in our CI yamls. Since this issue is specifically about CI failures, it has been fully resolved by #1594
